lib/jupyter/actions.js

```diff
@@ -381,6 +381,34 @@ define([
         );
       }
     }, 'extend-selection-below', 'vim-binding');
+
+    var cell_marker = {};
+
+    actions.register({
+      'handler': function(env, event) {
+        env.notebook.keyboard_manager.enabled = false;
+        document.addEventListener('keydown', function (event) {
+          var key = event.key;
+          var idx = env.notebook.get_selected_index();
+          cell_marker[key] = idx;
+          env.notebook.keyboard_manager.enabled = true;
+        }, { once: true });
+      }
+    }, 'mark-cell', 'vim-binding-normal');
+
+    actions.register({
+      'handler': function(env, event) {
+        env.notebook.keyboard_manager.enabled = false;
+        document.addEventListener('keydown', function (event) {
+          var key = event.key, idx;
+          if ((idx = cell_marker[key]) == null) return;
+          env.notebook.select(idx);
+          env.notebook.get_cell(idx).focus_cell();
+          env.notebook.scroll_cell_percent(idx, 0, 0);
+          env.notebook.keyboard_manager.enabled = true;
```

Comment:
Must enable `keyboard_manager` first if you use early return.

```suggestion
          env.notebook.keyboard_manager.enabled = true;
          var key = event.key, idx;
          if ((idx = cell_marker[key]) == null) return;
          env.notebook.select(idx);
          env.notebook.get_cell(idx).focus_cell();
          env.notebook.scroll_cell_percent(idx, 0, 0);
```